tests/migrations/env.py

```diff
@@ -0,0 +1,136 @@
+import asyncio
```

Comment:
shouldn't we use the actual `env.py`? because if someone will change the original one, the tests won't "test" the real case.